pkg/controller/elasticsearch/certificates/transport/reconcile.go

```diff
@@ -89,20 +89,20 @@ func reconcileNodeSetTransportCertificatesSecrets(
 	es esv1.Elasticsearch,
 	ssetName string,
 	rotationParams certificates.RotationParams,
-) error {
+) *reconciler.Results {
```

Comment:
This "bugfix" leads to the clusters constantly being in state `ApplyingChanges` because we always have a requeue 🤔